tests/e2e/specs/shopper/active-filters.test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Why is this needed? Because Puppeteer isn't in our `devDependencies`?

Reply:
Puppeteer isn't in our devDependencies because it is installed by `wordpress/e2e-tests` package.

Reply:
Mm, I think that as long as Puppeteer is a peer dependency of another package, that's fine. But since we are now importing something from it, I believe this should be added to the `devDependencies`, what do you think?

Like, if `wordpress/e2e-tests` dependencies change (even in version) this might cause errors in this code. Since we are importing directly, I think it now becomes an internal dep.